index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <!-- fontawesme link -->
+    <script src="https://kit.fontawesome.com/fc7e7ce012.js" crossorigin="anonymous"></script>
+    <!-- style css -->
+    <link rel="stylesheet" href="style.css">
+
+
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="navbar">
+                <ul class="top-navbar-menu">
+                    <li><a href="">Facebook</a></li>
+                    <li><a href="">Twitter</a></li>
+                    <li><a href="">English</a></li>
+                    <li><a href="">My Page</a></li>
+                    <li><a href="">Log out</a></li>
+                </ul>
+                <div class="bottom-navbar">
+                    <a class="logo" href="index.html">Logo</a>
+                    <ul class="bottom-navbar-menu">
+                        <li><a href="about.html">About</a></li>
+                        <li><a href="">Program</a></li>
+                        <li><a href="">Join</a></li>
+                        <li><a href="">Sponsor</a></li>
+                        <li><a href="">News</a></li>
+                        <li><a href="">Campiagn</a></li>
+                    </ul>   
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
```

Comment:
- [ ] As the previous reviwer said the HTML structure of your page can be better. Since you have a main tag as the main container of the content of the website. Inside this main tag, semantically all the three sections, the Headline, Main program section, and the Featured speakers section should use a section tag. below 👇🏾 is how you should structure your HTML to be semantically correct.
````
<main>
  <header>
    Header
  </header>
  <section>
    main program content is placed here
    <ul>
      <li><article>Lecture</article></li>
      <li><article>exhibition</article></li>
      <li><article>Forum</article></li>
      <li><article>workshop</article></li>
      <li><article>Tour</article></li>      
    </ul>
  </section>
  <section>
    <ul>
      <li><article>Speaker one</article></li>
      <li><article>Speaker two</article></li>
      <li><article>Speaker three</article></li>
      <li><article>Speaker four</article></li>
      <li><article>Speaker five</article></li>
      <li><article>Speaker six</article></li>
    </ul>
  </section>
  <section>
    partner section content is placed here
  </section>
  <footer>
    footer section content is placed here
  </footer>
</main>
````
A similar thing applies to your About page HTML structure.
I wish you the best.